tests/isar/models/test_start_mission_definition.py

```diff
@@ -51,3 +59,36 @@ def test_duplicate_id_check(
     duplicates: List[str] = get_duplicate_ids(item_list)
     has_duplicates: bool = len(duplicates) > 0
     assert has_duplicates == expected_boolean
+
+
+def test_mission_definition_to_isar_mission():
+    dirname = os.path.dirname(__file__)
+    filepath = os.path.join(dirname, "example_mission_definition.json")
+
+    with open(filepath) as f:
+        datax = json.load(f)
+        mission_definition = StartMissionDefinition(**datax)
+
+    generated_mission: Mission = to_isar_mission(mission_definition)
+    assert generated_mission.id == "generated_mission_id"
+    assert generated_mission.name == "my-mission"
+    assert len(generated_mission.tasks) == 1
+
+    task = generated_mission.tasks[0]
+    assert task.id == "generated_task_id"
+    assert task.tag_id == "MY-TAG-123"
+
+    first_step = task.steps[0]
+    # assert first_step.id == "generated_inspection_id"
```

Comment:
Do these fail?

Reply:
Yes, line 82 would fail, therefore it is commented out for now. It might be desired behavior, since the .id attribute will be set by the constructor.